html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm

```diff
@@ -63,7 +63,12 @@ has_field 'id' =>
    type => 'Text',
    label => 'Profile Name',
    required => 1,
-   apply => [ { check => qr/^[a-zA-Z0-9][a-zA-Z0-9\._-]*$/ } ],
+   apply => [
+       {   check => qr/^[a-zA-Z0-9][a-zA-Z0-9\._-]*$/,
+           message =>
+             'An invalid character in profile id. The profile id can only contain alphanumeric characters, dashes, period and or underscores'
```

Comment:
Grammar is not right

Suggestions :
An invalid character was found....
Found an invalid character...